hive/src/test/java/com/esri/hadoop/hive/serde/TestEsriJsonSerDe.java

```diff
@@ -57,6 +59,49 @@ public void TestIntWrite() throws Exception {
 		Assert.assertEquals(7, jn.getIntValue());
 	}
 
+	@Test
+	public void TestEpochWrite() throws Exception {
+        ArrayList<Object> stuff = new ArrayList<Object>();
+		Properties proptab = new Properties();
+		proptab.setProperty(HiveShims.serdeConstants.LIST_COLUMNS, "when");
+		proptab.setProperty(HiveShims.serdeConstants.LIST_COLUMN_TYPES, "date");
+		SerDe jserde = mkSerDe(proptab);
+        StructObjectInspector rowOI = (StructObjectInspector)jserde.getObjectInspector();
+
+        // {"attributes":{"when":147147147147}}
+        long epoch = 147147147147L;
+		java.sql.Date expected = new java.sql.Date(epoch);
```

Comment:
Inconsistency: TestGeoJsonSerDe adjusts for time zone, but TestEsriJsonSerDe did not.